src/usage/usage.service.spec.ts

```
@@ -13,3 +26,549 @@ describe('calculateResolutionRate', () => {
+    return this;
+  }
+  // FOR UPDATE 잠금 select은 .for()로 끝난다(await 대상).
+  for(_strength: unknown, _config?: unknown): Promise<unknown[]> {
+    this.store.forUpdateCalls += 1;
+    return Promise.resolve([]);
+  }
+  // source scan select은 .then으로 await 된다.
+  then<TResult1 = SourceRow[], TResult2 = never>(
+    onfulfilled?:
+      | ((value: SourceRow[]) => TResult1 | PromiseLike<TResult1>)
+      | null,
```

ESLint

[error] Replace `|·((value:·SourceRow[])·=>·TResult1·|·PromiseLike<TResult1>)⏎·····` with `((value:·SourceRow[])·=>·TResult1·|·PromiseLike<TResult1>)`

(prettier/prettier)